account_move_default_journal/models/account_move.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021-2023 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model
+    def _search_default_journal(self, journal_types):
+        company_id = self._context.get("default_company_id", self.env.company.id)
+        company = self.env["res.company"].browse(company_id)
+        domain = [
```

Comment:
You do not need this domain, so delete setting it.

Reply:
AH I forgot to remove that yes.